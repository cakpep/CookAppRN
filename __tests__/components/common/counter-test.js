import React from 'react';
import { StyleSheet } from 'react-native';
import renderer from 'react-test-renderer';

import { MonoText } from '../../../src/components/common/StyledText';

test('render with styles', () => {
    
    const styles = StyleSheet.create({
        textHighlightContainer: {
            backgroundColor: '#fff',
            borderRadius: 3,
            padding: 4,
        }
    })

    const tree = renderer
        .create(<MonoText style = {styles.textHighlightContainer}> Test </MonoText>)
        .toJSON()
    expect(tree).toMatchSnapshot()
})

test('render with no styles', () => {
    const tree = renderer
        .create(<MonoText> No Styles </MonoText>)
        .toJSON()
    expect(tree).toMatchSnapshot()
})

test('it should be failed', () => {
    expect(1).toBe(2)
});
