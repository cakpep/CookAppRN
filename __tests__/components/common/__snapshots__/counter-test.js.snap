// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render with no styles 1`] = `
<Text
  style={
    Array [
      undefined,
      Object {
        "fontFamily": "space-mono",
      },
    ]
  }
>
   No Styles 
</Text>
`;

exports[`render with styles 1`] = `
<Text
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "borderRadius": 3,
        "padding": 4,
      },
      Object {
        "fontFamily": "space-mono",
      },
    ]
  }
>
   Test 
</Text>
`;
